src/xdocs/config_coding.xml

```diff
@@ -4987,6 +4987,117 @@ enum NoSemicolon {
       </subsection>
     </section>
 
+    <section name="UnnecessarySemicolonInTryWithResources">
+      <p>Since Checkstyle 8.22</p>
+      <subsection name="Description" id="UnnecessarySemicolonInTryWithResources_Description">
+        <p>
+            Checks if unnecessary semicolon is used in last resource declaration.
+        </p>
+      </subsection>
+
+      <subsection name="Properties" id="UnnecessarySemicolonInTryWithResources_Properties">
+            <table>
```

Comment:
please indent as +2 in whole new section.

Reply:
done